### PR TITLE
Merge English and Japanese glossaries into one

### LINKED ISSUE #60: Re: Merging en & ja jlreq glossary
I thought it would be easier to handle the remaining work related to this by opening an issue.

### PR DESCRIPTION
**Pull request by xfq**
Fix #60 using the method in https://github.com/w3c/jlreq/issues/60#issuecomment-503525335

Two things I haven't done yet:

* Translate the word "Transliteration" into Japanese
* Add JavaScript to sort by the first two columns

IMO the second point can be done after the merge.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/jlreq/pull/88.html" title="Last updated on Jul 1, 2019, 6:26 AM UTC (6ae9a5a)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/jlreq/88/1b85b17...6ae9a5a.html" title="Last updated on Jul 1, 2019, 6:26 AM UTC (6ae9a5a)">Diff</a>

### CODE REVIEW

**Comment by xfq** [comment]
Since PR Preview failed to build, people who would like to see the preview can go to https://raw.githack.com/w3c/jlreq/xfq/merge-glossary/index.html#appendix_7

**Comment by himorin** [comment]
@kidayasuo this PR is which I've mentioned in today's F2F.

**Review by r12a — APPROVED** [review]

**Comment by xfq** [comment]
I think we should translate the word "Transliteration" in the header into Japanese before merging this PR, at least. (Suggestions are welcome.)

**Comment by xfq** [comment]
Also, if we decide to merge this PR, I can help resolving potential merge conflicts for other glossary-related PRs (such as #59, #75, #87).

**Review by kidayasuo — APPROVED** [review]
approving the PR. Thank you for doing the merge!

**Comment by himorin** [comment]
> I think we should translate the word "Transliteration" in the header into Japanese before merging this PR, at least. (Suggestions are welcome.)

I think "よみ" as current separated version is fine. 

> Also, if we decide to merge this PR, I can help resolving potential merge conflicts for other glossary-related PRs (such as #59, #75, #87).

I'd close #59 and add new PR for that after this merged (I can work). For #75 and #87, yes, we need to deal with after this merged.

**Review by himorin — APPROVED** [review]

**Comment by xfq** [comment]
> > I think we should translate the word "Transliteration" in the header into Japanese before merging this PR, at least. (Suggestions are welcome.)
> 
> I think "よみ" as current separated version is fine.

Thanks @himorin. I updated the PR.

**Comment by r12a** [comment]
@xfq what's the outlook for you to build in the other PR changes mentioned above and merge this?

**Comment by xfq** [comment]
@r12a Sorry, I'm not sure I understand what you mean. Would you please explain?

**Comment by r12a** [comment]
Sorry about that. Let me try again: How soon will we merge this pull request?  And will you add to it the fixes for the other PRs mentioned in https://github.com/w3c/jlreq/pull/88#issuecomment-505466680 ?

**Comment by xfq** [comment]
Since I have three r+ now, I'll go ahead and merge this PR, and:

* File an issue about adding JavaScript to sort by the first two columns
* Update #75 and #87

Thanks for the review!
